modules/k8s-nmstate-deploying-nmstate-CLI.adoc

```diff
@@ -0,0 +1,101 @@
+// This is included in the following assemblies:
+//
+// networking/k8s_nmstate/k8s-nmstate-about-the-kubernetes-nmstate-operator.adoc
+
+:_content-type: PROCEDURE
+[id="installing-the-kubernetes-nmstate-operator-CLI_{context}"]
+= Installing the Kubernetes NMState Operator using the CLI
+Instead of using the web console, you can install the Kubernetes NMState Operator by using the {product-title} CLI (oc).
+
+.Prerequisites
+
+* Install the OpenShift CLI (`oc`).
+
+* Log in as a user with `cluster-admin` privileges.
+
+.Procedure
+
+. Create the `nmstate` operator namespace:
++
+[source,terminal]
```

Comment:
[source,yaml]

Reply:
So for these, I was also confused how to tag the codeblocks. The SME gave me this [MetalLB Operator cli install](https://docs.openshift.com/container-platform/4.9/networking/metallb/metallb-operator-install.html#nw-metallb-installing-operator-cli_metallb-operator-install) as a reference for this nmstate Operator, and in the adoc file these yaml looking pieces are [source,terminal], not yaml. So...I did the same. There is the cat command piece too, so maybe that's why its terminal?

Reply:
You're right.